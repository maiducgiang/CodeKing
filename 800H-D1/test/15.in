10
89
12 63 39 38 4 89 63 11 35 1 59 64 47 27 32 12 56 27 30 10 42 50 12 64 88 66 38 37 76 38 77 17 54 74 14 9 16 40 47 6 77 30 60 21 44 25 36 71 2 74 18 29 45 82 89 15 7 25 53 84 25 60 26 12 22 80 47 55 89 8 44 38 84 44 78 84 9 83 83 7 21 55 7 67 11 59 16 70 59 
100
7 21 14 83 67 1 86 35 22 43 53 30 65 69 42 76 31 84 21 80 6 24 29 17 60 48 88 27 28 35 80 29 19 7 81 95 61 7 20 90 76 85 14 21 90 43 58 16 9 27 90 82 84 5 41 85 99 11 72 99 53 82 11 65 59 61 77 26 93 98 30 68 27 70 65 95 58 64 3 49 50 60 9 61 13 72 86 41 78 82 20 64 1 32 36 88 44 92 95 51 
88
33 27 36 62 64 88 33 62 26 82 33 46 57 83 23 79 44 71 63 22 6 78 4 33 60 85 39 87 61 24 35 36 82 63 38 44 41 56 38 67 12 51 68 29 27 2 18 81 39 59 31 7 21 53 5 45 56 23 31 65 88 68 58 43 74 13 80 35 12 2 73 51 80 65 74 10 48 16 67 32 4 60 59 28 75 35 24 1 
96
33 23 32 71 53 31 60 39 26 55 7 62 17 59 46 75 95 50 47 96 83 75 44 25 36 35 56 92 9 22 57 73 57 63 3 55 6 37 59 13 76 47 93 95 90 96 37 25 81 5 45 86 61 32 1 70 64 91 21 32 95 22 22 5 79 22 59 40 12 41 95 28 61 45 6 45 57 28 85 60 62 64 88 51 16 86 68 69 95 87 25 23 50 8 2 47 
98
41 67 88 33 33 61 37 24 20 9 78 62 32 68 9 49 51 30 83 82 15 94 91 91 3 84 9 84 84 57 86 62 34 70 95 38 35 71 52 28 56 56 14 18 20 6 33 80 93 66 79 28 31 60 62 44 55 42 1 49 55 67 93 25 52 59 45 63 48 58 92 89 30 38 81 40 69 14 72 23 62 34 12 96 16 38 22 78 56 57 42 6 22 86 68 41 32 73 
90
63 10 76 1 78 15 84 82 38 24 12 62 36 64 65 25 26 38 53 19 33 60 8 44 10 52 24 8 53 55 38 77 32 59 63 45 39 32 80 52 46 89 26 74 73 56 80 59 3 9 49 3 61 12 45 42 64 46 66 32 10 11 72 54 9 6 4 23 35 71 68 26 56 41 22 22 75 45 87 31 27 50 6 81 46 41 5 34 20 62 
76
38 41 26 13 35 73 57 75 23 5 70 15 64 54 15 6 15 47 9 75 38 65 41 19 68 27 6 13 53 35 5 17 5 12 73 53 26 46 22 19 66 54 71 55 35 62 35 28 70 36 27 16 2 63 49 23 1 6 2 44 64 47 44 2 32 64 14 72 7 58 43 33 61 71 70 45 
78
63 78 51 42 30 14 28 16 50 8 32 3 65 21 38 62 48 71 69 9 32 60 12 55 4 62 10 8 19 1 78 71 30 43 46 76 70 78 47 68 40 17 11 75 49 28 72 21 4 6 47 53 14 18 17 45 15 3 38 37 51 57 47 71 67 7 43 29 25 16 48 40 49 15 69 47 33 56 
71
36 57 55 64 12 60 27 61 47 25 51 67 15 19 25 54 69 37 39 41 23 8 5 21 42 29 34 54 54 56 20 38 29 56 12 3 20 53 17 29 71 29 40 25 19 37 25 36 11 71 44 27 62 53 60 43 5 59 18 29 66 7 1 22 1 26 52 64 18 67 29 
94
77 55 92 45 66 3 58 58 41 20 25 85 49 92 39 37 8 47 30 40 16 55 72 8 31 8 93 83 68 34 22 3 32 38 42 84 11 92 52 7 22 38 62 10 83 14 1 31 92 9 25 22 62 36 62 36 1 1 30 21 5 58 37 77 13 40 27 27 25 67 62 71 2 78 30 59 72 88 50 51 57 79 85 15 12 45 64 54 64 42 64 26 76 7 
