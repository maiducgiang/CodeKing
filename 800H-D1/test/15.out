31
32
28
51
53
46
31
38
22
45
